WalletWasabi/Tor/Http/Extensions/HttpResponseMessageExtensions.cs

```diff
@@ -61,6 +66,23 @@ public static async Task ThrowRequestExceptionFromContentAsync(this HttpResponse
 			if (me.Content is { })
 			{
 				var contentString = await me.Content.ReadAsStringAsync(cancellationToken).ConfigureAwait(false);
+				var error = JsonConvert.DeserializeObject<Error>(contentString, new JsonSerializerSettings()
```

Comment:
What happens if it cannot deserialize? If it throws then we will never get to the line of further processing here: https://github.com/zkSNACKs/WalletWasabi/pull/5767/files#diff-692d4a4f7d2eeacc2012f438873c1b1d2e98cb92db69dd0592df35644d71c96fR88

Reply:
It doesn't throw because the `Error` handler ignores any problem and makes sure to return `null` in case the deserialization is not possible.